### PR TITLE
Add Chinese README

### PR DESCRIPTION
**Pull request by PlexPt**

### CODE REVIEW

**Comment by AntonOsika** [comment]
Thanks!

How do I know this is the correct translation if I don’t know Chinese? Did you use a tool or translate manually ?

File: README-CN.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+- 检查生成的文件结果`projects/my-new-project/workspace`
+
+## 特性
+
+您可以通过编辑文件夹中的文件来指定AI代理的“身份” `preprompts`。
+
+目前，编辑`preprompts`以及改进编写项目提示的方式是让代理记住项目之间的事情的方法。
+
+中的每个步骤`steps.py`都会将其与 GPT4 的通信历史记录存储在日志文件夹中，并且可以使用`scripts/rerun_edited_message_logs.py`.
```
Comment (JinchuLi2002) [inline README-CN.md:75]: This sentence is incomplete

File: README-CN.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+- `export OPENAI_API_KEY=[your api key]`
+
+**运行**：
+
+- 创建一个空文件夹。如果在存储库内，您可以运行：
+
+  - `cp -r projects/example/ projects/my-new-project`
+
+- 填写`prompt`新文件夹中的文件
```
Comment (JinchuLi2002) [inline README-CN.md:55]: this makes ppl think 'prompt' is a folder yet in reality it should be a file

**Review by JinchuLi2002 — CHANGES_REQUESTED** [review]
The translation looks a bit rough overall, is it machine translated?

**Comment by AntonOsika** [comment]
I'll close this, will not be able to keep this up to date :/

**Comment by AntonOsika** [comment]
Thanks for review @JinchuLi2002

**Pull request closed without merging**